### ISSUE TITLE
Frame wise Labels for Action Classification

### ISSUE THREAD

**Issue by SateeshKumar21** [IkeaASM/IKEA_ASM_Dataset#2]
Hi,

I wanted to extract frame-wise numpy labels for each video. I came across the "get_action_labels_from_json" function in the the script IkeaActionDataset.py. But, this function yields incorrect number of labels in some cases. For instance, the video: 0039_white_floor_08_04_2019_08_28_10_40 has 1934 frames but the number of labels extracted for this video via the aforementioned function is 1932.

I have used your extract_frames script for converting videos into frames.

Can you please let me know what may be causing the discrepancy in this case?. Will really appreciate your help. Alternatively, please suggest some other way for extracting frame-wise labels.

**Comment by sitzikbs** [comment]
Hi,
Which device are you extracting the frames from? (I assume you are using device 3)

When the data was captured, there were 3 devices capturing simultaneously. In a few cases, when finishing the acquisition there was a small lag between the devices, resulting in some videos being a few frames longer than the others. 
When labelling, we made sure that all 3 videos have the same number of frames (taking the minimum number of frames from all 3). 

In short, you can ignore the last frames alternatively, you can get the labelled number of frames from the DB file or extract the number of frames from other devices as the minimum of all 3. 

It might be possible to manually delete them (since it really is only a handful of cases).

**Comment by SateeshKumar21** [comment]
Okay, Got it.

Thanks a lot.
